x-pack/plugins/security_solution/public/common/components/exceptions/builder/exception_item.tsx

```diff
@@ -0,0 +1,141 @@
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useMemo } from 'react';
+import { EuiButtonIcon, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import styled from 'styled-components';
+
+import { Entry, EntriesArray } from '../../../../../public/lists_plugin_deps';
+import { IIndexPattern } from '../../../../../../../../src/plugins/data/common';
+import { AndOrBadge } from '../../and_or_badge';
+import { EntryItemComponent } from './entry_item';
+import { getFormattedBuilderEntries } from '../helpers';
+import { FormattedBuilderEntry, ExceptionsBuilderExceptionItem } from '../types';
+
+const MyInvisibleAndBadge = styled(EuiFlexItem)`
+  visibility: hidden;
+`;
+
+const MyFirstRowContainer = styled(EuiFlexItem)`
+  padding-top: 20px;
+`;
+
+interface ExceptionListItemProps {
+  exceptionItem: ExceptionsBuilderExceptionItem;
+  exceptionId: string;
+  exceptionItemIndex: number;
+  isLoading: boolean;
+  indexPattern: IIndexPattern;
+  andLogicIncluded: boolean;
+  onDeleteExceptionItem: (item: ExceptionsBuilderExceptionItem, index: number) => void;
+  onExceptionItemChange: (item: ExceptionsBuilderExceptionItem, index: number) => void;
+}
+
+export const ExceptionListItemComponent = React.memo<ExceptionListItemProps>(
+  ({
+    exceptionItem,
+    exceptionId,
+    exceptionItemIndex,
+    indexPattern,
+    isLoading,
+    andLogicIncluded,
+    onDeleteExceptionItem,
+    onExceptionItemChange,
+  }) => {
+    const handleEntryChange = (entry: Entry, entryIndex: number): void => {
+      const updatedEntries: EntriesArray = [
+        ...exceptionItem.entries.slice(0, entryIndex),
+        { ...entry },
+        ...exceptionItem.entries.slice(entryIndex + 1),
+      ];
+      const updatedExceptionItem: ExceptionsBuilderExceptionItem = {
+        ...exceptionItem,
+        entries: updatedEntries,
+      };
+      onExceptionItemChange(updatedExceptionItem, exceptionItemIndex);
+    };
+
+    const handleDeleteEntry = (entryIndex: number): void => {
+      const updatedEntries: EntriesArray = [
+        ...exceptionItem.entries.slice(0, entryIndex),
+        ...exceptionItem.entries.slice(entryIndex + 1),
+      ];
+      const updatedExceptionItem: ExceptionsBuilderExceptionItem = {
+        ...exceptionItem,
+        entries: updatedEntries,
+      };
+
+      onDeleteExceptionItem(updatedExceptionItem, exceptionItemIndex);
+    };
+
+    const entries = useMemo(
+      (): FormattedBuilderEntry[] =>
+        indexPattern != null ? getFormattedBuilderEntries(indexPattern, exceptionItem.entries) : [],
+      [indexPattern, exceptionItem.entries]
+    );
+
+    const andBadge = useMemo((): JSX.Element => {
+      const badge = <AndOrBadge includeAntennas type="and" />;
+      if (entries.length > 1 && exceptionItemIndex === 0) {
+        return <MyFirstRowContainer grow={false}>{badge}</MyFirstRowContainer>;
```

Comment:
The `and` badge antennas expand the height of the parent container. Since the first row includes the labels, the antennas were expanded beyond just the exception item entries. This added padding fixes that.